pkg/envinfo/envinfo.go

```diff
@@ -171,6 +186,8 @@ func (esi *EnvSpecificInfo) SetConfiguration(parameter string, value interface{}
 		case "push":
 			pushCommandValue := value.(EnvInfoPushCommand)
 			esi.componentSettings.PushCommand = &pushCommandValue
+		case "appname":
+			esi.componentSettings.AppName = parameter
```

Comment:
Dont see this being used anywhere, just curious.

also `esi.componentSettings.AppName = parameter` doesn't look right. We should be using `value`?

Reply:
removed